diff -urN pcmcia-cs-3.2.0.org/man/Makefile pcmcia-cs-3.2.0/man/Makefile
--- pcmcia-cs-3.2.0.org/man/Makefile	Thu Aug  8 22:40:58 2002
+++ pcmcia-cs-3.2.0/man/Makefile	Thu Aug  8 22:41:56 2002
@@ -28,21 +28,21 @@
 
 install-man1-x11: $(MAN1_X11)
 ifdef HAS_FORMS
-	@mkdir -p $(XMANDIR)/man1
-	cp $(MAN1_X11) $(XMANDIR)/man1
+	@install -d $(XMANDIR)/man1
+	install -m644 $(MAN1_X11) $(XMANDIR)/man1
 endif
 
 install-man4: $(MAN4)
-	@mkdir -p $(MANDIR)/man4
-	cp *.4 $(MANDIR)/man4
+	@install -d $(MANDIR)/man4
+	install -m644 *.4 $(MANDIR)/man4
 
 install-man5: $(MAN5)
-	@mkdir -p $(MANDIR)/man5
-	cp *.5 $(MANDIR)/man5
+	@install -d $(MANDIR)/man5
+	install -m644 *.5 $(MANDIR)/man5
 
 install-man8: $(MAN8)
-	@mkdir -p $(MANDIR)/man8
-	cp *.8 $(MANDIR)/man8
+	@install -d $(MANDIR)/man8
+	install -m644 *.8 $(MANDIR)/man8
 
 install: install-man1-x11 install-man4 install-man5 install-man8
 
