diff -Naur pcmcia-cs-3.1.29/etc/config pcmcia-cs-3.1.29-p/etc/config
--- pcmcia-cs-3.1.29/etc/config	Mon Aug 27 02:26:02 2001
+++ pcmcia-cs-3.1.29-p/etc/config	Thu Oct  4 12:58:09 2001
@@ -1505,11 +1505,11 @@
   cis "cis/DP83903.dat"
   bind "pcnet_cs" to 0, "serial_cs" to 1
 
-card "NSC DP83903-based Ethernet/Modem"
-  #version "Multifunction Card", "LAN and Modem 336", "Multifunction"
-  manfid 0x0175, 0x0000
-  cis "cis/DP83903.dat"
-  bind "pcnet_cs" to 0, "serial_cs" to 1
+#card "NSC DP83903-based Ethernet/Modem"
+#  #version "Multifunction Card", "LAN and Modem 336", "Multifunction"
+#  manfid 0x0175, 0x0000
+#  cis "cis/DP83903.dat"
+#  bind "pcnet_cs" to 0, "serial_cs" to 1
 
 card "Ositech Jack of Diamonds Ethernet/Modem"
   version "Ositech", "Trumpcard:Jack of Diamonds Modem+Ethernet"
