diff -urN pcmcia-cs-3.1.30.org/man/Makefile pcmcia-cs-3.1.30/man/Makefile
--- pcmcia-cs-3.1.30.org/man/Makefile	Thu May 16 15:32:51 2002
+++ pcmcia-cs-3.1.30/man/Makefile	Thu May 16 15:37:31 2002
@@ -28,19 +28,19 @@
 
 install-man1-x11: $(MAN1_X11)
 	@mkdir -p $(XMANDIR)/man1
-	cp $(MAN1_X11) $(XMANDIR)/man1
+	install -m644 $(MAN1_X11) $(XMANDIR)/man1
 
 install-man4: $(MAN4)
 	@mkdir -p $(MANDIR)/man4
-	cp *.4 $(MANDIR)/man4
+	install -m644 *.4 $(MANDIR)/man4
 
 install-man5: $(MAN5)
 	@mkdir -p $(MANDIR)/man5
-	cp *.5 $(MANDIR)/man5
+	install -m644 *.5 $(MANDIR)/man5
 
 install-man8: $(MAN8)
 	@mkdir -p $(MANDIR)/man8
-	cp *.8 $(MANDIR)/man8
+	install -m644 *.8 $(MANDIR)/man8
 
 install: install-man1-x11 install-man4 install-man5 install-man8
 
diff -urN pcmcia-cs-3.1.30.org/wavelan2_cs.mk pcmcia-cs-3.1.30/wavelan2_cs.mk
--- pcmcia-cs-3.1.30.org/wavelan2_cs.mk	Thu May 16 15:32:51 2002
+++ pcmcia-cs-3.1.30/wavelan2_cs.mk	Thu May 16 15:33:01 2002
@@ -24,7 +24,6 @@
 
 install:
 	$(MAKE) -C wireless install MODULES=wavelan2_cs.o
-	/sbin/depmod -a
 	$(MAKE) -C man install-man4 MAN4=wavelan2_cs.4
 	$(MAKE) -C etc -f wavelan2_cs.mk $@
 
