diff -urN pcmcia-cs-3.2.7.org/Configure pcmcia-cs-3.2.7/Configure
--- pcmcia-cs-3.2.7.org/Configure	2004-01-02 18:16:40.979224627 +0100
+++ pcmcia-cs-3.2.7/Configure	2004-01-02 18:17:53.634117043 +0100
@@ -138,6 +138,8 @@
     --bsd)		SYSV_INIT=n		;;
     --rcdir=*)		arg RC_DIR "$1"		;;
     --extraver=*)	arg EXTRAVER "$1"	;;
+    --smp)              CONFIG_SMP=y            ;;
+    --up)               CONFIG_SMP=n            ;;
     *)			usage			;;
     esac
     shift
@@ -529,6 +531,7 @@
     printflag "Symmetric multiprocessing support" CONFIG_SMP
     if [ "$CONFIG_SMP" = "y" ] ; then
 	echo "#define __SMP__ 1" >> $CONFIG_H
+	echo "#define __KERNEL_SMP 1" >> $CONFIG_H
 	if [ -n "$CONFIG_NR_CPUS" ] ; then
 	    write_int CONFIG_NR_CPUS
 	fi
@@ -713,12 +716,6 @@
 	echo "CKSUM=\"`cksum < $AUTOCONF`\"" >> $CONFIG
 	echo "#define CONFIG_MODULES 1" >> $CONFIG_H
 	configcheck PCMCIA
-	if qgrep "^ *SMP *= *1" $LINUX/Makefile ; then
-	    CONFIG_SMP=y
-	else
-	    configcheck SMP
-	    eval `grep "^CONFIG_NR_CPUS=" $AUTOCONF`
-	fi
 	if [ $VERSION_CODE -lt `version 2 1 7` ] ; then
 	    configcheck BLK_DEV_IDE_PCMCIA
 	fi
