--- pcmcia-cs-3.2.5/Configure.orig	2003-08-13 08:26:23.000000000 +0200
+++ pcmcia-cs-3.2.5/Configure	2003-10-18 17:35:19.000000000 +0200
@@ -135,6 +135,8 @@
     --bsd)		SYSV_INIT=n		;;
     --rcdir=*)		arg RC_DIR "$1"		;;
     --extraver=*)	arg EXTRAVER "$1"	;;
+    --smp)		CONFIG_SMP=y		;;
+    --up)		CONFIG_SMP=n		;;
     *)			usage			;;
     esac
     shift
@@ -526,6 +528,7 @@
     printflag "Symmetric multiprocessing support" CONFIG_SMP
     if [ "$CONFIG_SMP" = "y" ] ; then
 	echo "#define __SMP__ 1" >> $CONFIG_H
+	echo "#define __KERNEL_SMP 1" >> $CONFIG_H
     fi
     printflag "Preemptive kernel support" CONFIG_PREEMPT
     printflag "High memory support" CONFIG_HIGHMEM
@@ -687,11 +690,6 @@
 	echo "CKSUM=\"`cksum < $AUTOCONF`\"" >> $CONFIG
 	echo "#define CONFIG_MODULES 1" >> $CONFIG_H
 	configcheck PCMCIA
-	if qgrep "^ *SMP *= *1" $LINUX/Makefile ; then
-	    CONFIG_SMP=y
-	else
-	    configcheck SMP
-	fi
 	if [ $VERSION_CODE -lt `version 2 1 7` ] ; then
 	    configcheck BLK_DEV_IDE_PCMCIA
 	fi
