diff -urN pcmcia-cs-3.2.6.org/Configure pcmcia-cs-3.2.6/Configure
--- pcmcia-cs-3.2.6.org/Configure	2003-12-08 21:32:13.381938693 +0100
+++ pcmcia-cs-3.2.6/Configure	2003-12-08 21:33:05.152143590 +0100
@@ -107,6 +107,9 @@
     exit 1
 }
 
+HAS_XAW=n
+HAS_GTK=n
+HAS_FORMS=n
 while [ $# -gt 0 ] ; do
     case "$1" in
     -n|--noprompt)	PROMPT=n		;;
@@ -140,6 +143,9 @@
     --extraver=*)	arg EXTRAVER "$1"	;;
     --smp)              CONFIG_SMP=y            ;;
     --up)               CONFIG_SMP=n            ;;
+    --has-xaw)          HAS_XAW=y               ;;
+    --has-gtk)          HAS_GTK=y               ;;
+    --has-forms)        HAS_FORMS=y             ;;
     *)			usage			;;
     esac
     shift
