diff -urN pcmcia-cs-3.2.3.org/cardmgr/Makefile pcmcia-cs-3.2.3/cardmgr/Makefile
--- pcmcia-cs-3.2.3.org/cardmgr/Makefile	Tue Feb 11 12:59:34 2003
+++ pcmcia-cs-3.2.3/cardmgr/Makefile	Tue Feb 11 13:07:41 2003
@@ -91,14 +91,14 @@
 	$(CC) $< -o $@ `gtk-config --libs`
 
 install-cardinfo: cardinfo
-	@mkdir -p $(PREFIX)/usr/X11R6/bin
-	cp -f cardinfo $(PREFIX)/usr/X11R6/bin
-	chmod u+s $(PREFIX)/usr/X11R6/bin/cardinfo
+	@mkdir -p $(PREFIX)/usr/bin
+	cp -f cardinfo $(PREFIX)/usr/bin
+	chmod u+s $(PREFIX)/usr/bin/cardinfo
 
 install-xcardinfo: xcardinfo
-	@mkdir -p $(PREFIX)/usr/X11R6/bin
-	cp -f xcardinfo $(PREFIX)/usr/X11R6/bin
-	chmod u+s $(PREFIX)/usr/X11R6/bin/xcardinfo
+	@mkdir -p $(PREFIX)/usr/bin
+	cp -f xcardinfo $(PREFIX)/usr/bin
+	chmod u+s $(PREFIX)/usr/bin/xcardinfo
 
 install-gpccard: gpccard
 	@mkdir -p $(PREFIX)/usr/bin
diff -urN pcmcia-cs-3.2.3.org/man/Makefile pcmcia-cs-3.2.3/man/Makefile
--- pcmcia-cs-3.2.3.org/man/Makefile	Tue Feb 11 12:59:34 2003
+++ pcmcia-cs-3.2.3/man/Makefile	Tue Feb 11 13:10:52 2003
@@ -28,8 +28,8 @@
 
 install-man1-x11: $(MAN1_X11)
 ifdef HAS_FORMS
-	@mkdir -p $(XMANDIR)/man1
-	cp $(MAN1_X11) $(XMANDIR)/man1
+	@mkdir -p $(MANDIR)/man1
+	cp $(MAN1_X11) $(MANDIR)/man1
 endif
 
 install-man4: $(MAN4)
