--- pcmcia-cs-3.1.30/cardmgr/cardinfo.c.orig	Wed Nov 14 02:24:36 2001
+++ pcmcia-cs-3.1.30/cardmgr/cardinfo.c	Sat Apr  5 21:31:54 2003
@@ -486,7 +486,7 @@
     if (setsid() < 0)
         perror("detaching from tty");
 
-#if (FL_REVISION >= 80)
+#if (FL_VERSION > 0) || (FL_REVISION >= 80)
     fl_flip_yorigin();
     fl_initialize(&argc, argv, "cardinfo", 0, 0);
 #else
@@ -523,7 +523,7 @@
     }
 
     event_log = fl_add_browser(FL_NORMAL_BROWSER, 10, 5, 270, 60, "");
-#if (FL_REVISION < 88)
+#if (FL_VERSION == 0) && (FL_REVISION < 88)
     fl_set_browser_leftslider(event_log, 1);
 #endif
     fl_set_browser_fontsize(event_log, FL_SMALL_SIZE);
