diff -urN pcmcia-cs-3.1.30.orig/etc/config pcmcia-cs-3.1.30/etc/config
--- pcmcia-cs-3.1.30.orig/etc/config	Sat Nov 16 21:54:46 2002
+++ pcmcia-cs-3.1.30/etc/config	Sat Nov 16 21:59:28 2002
@@ -117,6 +117,9 @@
 device "ohci1394_cb"
   class "ieee1394" module "cb_enabler", "ohci1394_cb"
 
+device "realtek_cb"
+    class "network" module "realtek_cb"
+
 #
 # Ethernet adapter definitions
 #
@@ -688,6 +691,10 @@
   cis "cis/tamarack.dat"
   bind "pcnet_cs"
 
+card "Surecom EP-428"
+    manfid 0x0000, 0x024c
+    bind "realtek_cb"
+
 card "TDK LAC-CD02x Ethernet"
   version "TDK", "LAC-CD02x"
   bind "fmvj18x_cs"
