diff -Nru pcmcia-cs-3.1.21/Configure pcmcia-cs-3.1.21.new/Configure
--- pcmcia-cs-3.1.21/Configure	Thu Aug 17 21:17:11 2000
+++ pcmcia-cs-3.1.21.new/Configure	Thu Oct 26 11:53:07 2000
@@ -795,39 +795,6 @@
 
 #=======================================================================
 
-# Optional stuff
-
-HAS_FORMS=n
-if [ -r /usr/include/X11/Xlib.h -o \
-     -r /usr/X11R6/include/X11/Xlib.h ] ; then
-    echo "X Windows include files found."
-    for f in /usr/{X11/,X11R6/,local/,}lib/libforms.{so,a} ; do
-	if [ -r $f ] ; then break ; fi
-    done
-    for g in /usr/{X11/,X11R6/,local/,}include/{,X11/}forms.h ; do
-	if [ -r $g ] ; then break ; fi
-    done
-    if [ -r $f -a -r $g ] ; then
-	echo "$f and $g found."
-	HAS_FORMS=y
-    else
-	echo "Forms library not installed."
-    fi
-else
-    echo "X Windows include files not installed."
-fi
-
-if [ "$HAS_FORMS" != "y" ] ; then
-    echo "    If you wish to build the 'cardinfo' control panel, you" \
-	 "need the Forms"
-    echo "    library and the X Windows include files.  See the HOWTO" \
-	 "for details."
-    HAS_FORMS=n
-fi
-write_bool HAS_FORMS
-
-#=======================================================================
-
 if [ ! -d /var/run ] ; then
     echo "WARNING: /var/run not found."
     echo "    To fix, do 'mkdir /var/run'."
diff -Nru pcmcia-cs-3.1.21/cardmgr/Makefile pcmcia-cs-3.1.21.new/cardmgr/Makefile
--- pcmcia-cs-3.1.21/cardmgr/Makefile	Mon Jun 12 23:33:01 2000
+++ pcmcia-cs-3.1.21.new/cardmgr/Makefile	Thu Oct 26 11:54:18 2000
@@ -28,13 +28,6 @@
 TOOLS += ide_info
 endif
 
-ifdef HAS_FORMS
-SRCS  += cardinfo.c
-CPPFLAGS += -I/usr/X11R6/include -I/usr/X11/include -I/usr/X11R6/include/X11
-EXTRA += cardinfo
-I_EXTRA += install-cardinfo
-endif
-
 ifdef CONFIG_ISA
 PROBE = probe
 ifdef PREFIX
